services/EventTokenService.js

```diff
@@ -0,0 +1,67 @@
+const jwt = require("jsonwebtoken");
+const uuid4 = require("uuid4");
+
+// A service class for Token generation and management
+class EventTokenService {
+  static #app_access_key = config.get("Event100ms.APP_ACCESS_KEY");
+  static #app_secret = config.get("Event100ms.APP_SECRET");
```

Comment:
will this work in production? production does not know what is `config`

Reply:
@prerana1821 we have to provide a fallback value for production by using or operator. 
Because this configs is only for local environment.